hack/observability/grafana/dashboards/cluster-api-state.json

```diff
@@ -564,1196 +564,6 @@
         "x": 0,
         "y": 5
       },
```

Comment:
~ Only moved some panels from this dashboard to the perf dashboard. This dashboard now looks like this

![pr-state](https://github.com/kubernetes-sigs/cluster-api/assets/4662360/bbfc7d9a-c11d-479e-b326-a113795340c2)